karma.conf.js

```diff
@@ -0,0 +1,2 @@
+/** @param {import('karma').Config} config */
+module.exports = config => config.set({ client: { captureConsole: false } });
```

Comment:
Is it needed?

Reply:
It just makes the terminal output clearer when running tests.